common/core/network.go

```diff
@@ -147,9 +147,15 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) (resultError error) {
 		// The network must be created beforehand
 		nwConfig, err = plugin.nm.GetNetworkByName(cniConfig.Name)
 
+		if err != nil {
```

Comment:
why not return err, instead of creating a custom error message? Or may be append err to the custom error message.